contribute/process/release-management.md

```diff
@@ -4,79 +4,156 @@ OpenEBS follows a monthly release cadence. The scope of the release is determine
+- openebs/jiva
+- openebs/libcstor
+- openebs/cstor
+- openebs/istgt
+- openebs/external-storage
+- openebs/maya
+- openebs/velero-plugin
+- openebs/cstor-csi
+- openebs/jiva-operator
+- openebs/jiva-csi
+- openebs/api
+- openebs/cstor-operators
+- openebs/upgrade
+
+The following repositories currently follow a custom release version.
+- openebs/node-disk-manager
+- openebs/zfs-localpv
+
+The following repositories are under active development and releases are created from master branch.
+- openebs/Mayastor
+- openebs/linux-utils
+- openebs/rawfile-localpv
+- openebs/monitor-pv
+
+To verify that release branches are created, you can run the following script:
+
+```
+git clone https://github.com/openebs/charts
+cd charts
+git checkout gh-pages
+
+cd scripts/release
+./check-release-branch.sh <release-branch>
+```
 
 ## Release Tagging
 
 OpenEBS Components are released as container images with versioned tags.
 
 OpenEBS components are spread across various repositories. Creating a release tag on the repository will trigger the build, integration tests and pushing of the docker image to [docker hub](https://hub.docker.com/u/openebs) and [quay](https://quay.io/organization/openebs/) container repositories.
 
-The format of the release tag is either "Release-Name-RC1" or "Release-Name" depending on whether the tag is a release candidate or a release. (Example: 1.0.0-RC1 is a GitHub release tag for OpenEBS release candidate build. 1.0.0 is the release tag that is created after the release criteria are satisfied by the release candidate builds.)
+The format of the GitHub release tag is either "Release-Name-RC1" or "Release-Name" depending on whether the tag is a release candidate or a release. (Example: v1.0.0-RC1 is a GitHub release tag for OpenEBS release candidate build. v1.0.0 is the release tag that is created after the release criteria are satisfied by the release candidate builds.)
+
+Each repository has the automation scripts setup to push the container images to docker and quay container repositories. The container image tag will be derived from GitHub release tag by truncating `v` from the release name. (Example: v1.0.0-RC1 will create container images for 1.0.0-RC1).
 
-Each repository has the automation scripts setup to push in the container images to docker and quay container repositories.
+Once a release made on a repository, Travis will trigger the release on the dependent repositories. The release tree looks as follows:
 
-The release tags are applied in the following order:
 - openebs/linux-utils
-- openebs/libcstor
-- openebs/cstor
-- openebs/istgt
-- openebs/jiva
+  - openebs/jiva
+  - openebs/libcstor
+    - openebs/cstor
+      - openebs/istgt
+        - openebs/cstor-operators
+        - openebs/external-storage
+          - openebs/maya
+            - openebs/velero-plugin
+            - openebs/cstor-csi
+              - openebs/jiva-operator
+                - openebs/jiva-csi
```

Comment:
Are the above indentations correct

Reply:
Thanks! jiva-operator gets tagged by maya. Fixed it.

Reply:
The rest are in the order, as they are tagged in current releases.